.github/ISSUE_TEMPLATE/02_feature_request.yml

```
@@ -0,0 +1,20 @@
+name: Feature Request
+description: File a feature request.
+labels: ["suggestion"]
```

```suggestion
labels: ["Suggestion"]
```
Same as the bug template